core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -26,6 +29,7 @@
     private final ScrollOffsetCalculator scrollOffsetCalculator;
     private final FastForwarder fastForwarder;
 
+    private Set<ScrollingPageChangeListener> onScrollingPageChangeListeners;
```

Comment:
`ScrollingPageChangeListener` doesn't have an `equals` method, so if you have two instances with the same dependencies, can they both be added? And would that be bad?